### PR TITLE
README.md: cleanup non-existent directories

### PR DESCRIPTION
**Pull request by artur-rs**

### CODE REVIEW

File: test-cases/test-cases.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Test cases
```
Comment (artur-rs) [inline test-cases/test-cases.md:1]: @m-iwanicki let's add reference to the Cukinia test code

File: README.md [hunk 2]
```diff
@@ -9,18 +9,13 @@ SPDX-License-Identifier: CC-BY-SA-4.0
...
 
-The `boards` directory contains the current test results retrieved from the
-[Dasharo Test & Feature Matrix](https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit#gid=736501945)
-sheet.
-
-The `archive` section contains archival test results retrieved from previous
-iterations of the test sheets and is preserved here for historical purposes.
+* [APU](./boards/APU/) - results for different PC Engines APU boards
 
 ## Links
```
Comment (artur-rs) [inline README.md:16]: @m-iwanicki let's add reference to the Cukinia test code

File: test-cases/test-cases.md (hunk 1 above)
Comment (m-iwanicki) [reply test-cases/test-cases.md:1]: Added in [dae787a](https://github.com/TrenchBoot/test-results/pull/1/commits/dae787af8aeb762353cfddc14e22674029c25e41).
Added link with exact commit, after it's merged to master I could change it to always point to newest version on master.

File: README.md (hunk 2 above)
Comment (m-iwanicki) [reply README.md:16]: @artur-rs Added in [dae787a](https://github.com/TrenchBoot/test-results/pull/1/commits/dae787af8aeb762353cfddc14e22674029c25e41).
Added link with exact commit, after it's merged to master I could change it to always point to newest version on master.

File: test-cases/test-cases.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    - SNB_IVB_SINIT*.bin
+    - TGL_SINIT*.bin
+10. `skl.bin` exists at `/boot/skl.bin`
+11. `grub.cfg` contains `/skl.bin` line
+12. `tpm2_tools` is installed - check if `tpm2_tools` package is installed
+13. `rsync` is installed - check if `rsync` is installed
+14. There is at least one active PCR bank - check if there is at least one
+directory: `/sys/class/tpm/tpm0/pcr-sha*`
+15. PCRs 0 to 23 can be read from `/sys/class/tpm/tpm0/pcr-sha*`
+16. When SKL booting - PCR 17 & 18 shouldn't contain only `0`s or only `F`s
```
Comment (krystian-hebel) [inline test-cases/test-cases.md]:
```suggestion
16. When booting with slaunch - PCR 17 & 18 shouldn't contain only `0`s or only `F`s
```

File: test-cases/test-cases.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Test cases
+
+Some of the test cases below are implemented in
+[cukinia](https://github.com/3mdeb/meta-trenchboot/blob/8b4812d8577c57b2ee9d1b6bf6898b84bb863fe5/recipes-tests/trenchboot-tests/trenchboot-tests/cukinia.conf).
+Currently, those are tests 5 through 18.
+
+1. UEFI boot works - TrenchBoot boots into OS
+2. Legacy boot works - TrenchBoot boots into OS
+3. Normal boot (`boot` menuentry) works - TrenchBoot boots into OS
+4. SKL boot (`skl-boot` menuentry) works - TrenchBoot boots into OS
```
Comment (krystian-hebel) [inline test-cases/test-cases.md]: This probably also needs change in grub.cfg:

```suggestion
4. Slaunch boot (`slaunch-boot` menuentry) works - TrenchBoot boots into OS
```

SKL is AMD only, but it should work botn on AMD and Intel.

File: README.md [hunk 5]
```diff
@@ -9,18 +9,15 @@ SPDX-License-Identifier: CC-BY-SA-4.0
...
 
-- [Dasharo Test Specification](https://docs.dasharo.com/unified-test-documentation/overview/)
+* [Test cases](./test-cases/) - contains test scenarios and cases
+* [Cukinia]([cukinia](https://github.com/3mdeb/meta-trenchboot/blob/8b4812d8577c57b2ee9d1b6bf6898b84bb863fe5/recipes-tests/trenchboot-tests/trenchboot-tests/cukinia.conf))
```
Comment (artur-rs) [inline README.md]:
```suggestion
* [Cukinia](https://github.com/3mdeb/meta-trenchboot/blob/8b4812d8577c57b2ee9d1b6bf6898b84bb863fe5/recipes-tests/trenchboot-tests/trenchboot-tests/cukinia.conf)
```

File: test-cases/test-cases.md (hunk 3 above)
Comment (m-iwanicki) [reply test-cases/test-cases.md]: fixed in 52c44ea2724ca1dccd8538032cc9232c0fd569d7

File: test-cases/test-cases.md (hunk 4 above)
Comment (m-iwanicki) [reply test-cases/test-cases.md]: fixed in 52c44ea2724ca1dccd8538032cc9232c0fd569d7. Also changed in grub.cfg

File: README.md (hunk 5 above)
Comment (m-iwanicki) [reply README.md]: 52c44ea2724ca1dccd8538032cc9232c0fd569d7

File: boards/APU/results.csv [hunk 6]
```diff
@@ -0,0 +1,29 @@
+Test name,APU2C
```
Comment (krystian-hebel) [inline boards/APU/results.csv:1]: This file needs to be recreated with renamed tests.

Also, I find it unlikely that `SKL boot  works,PASS` and ``PCR 17 doesn't contain only `0`s or only `F`s,FAIL`` at the same time.

Comment (m-iwanicki) [reply boards/APU/results.csv:1]: In the test case it's:
Slaunch boot (`slaunch-boot` menuentry) works - TrenchBoot boots into OS

Only checks that system booted from `slaunch-boot` menuentry.

Comment (m-iwanicki) [reply boards/APU/results.csv:1]: Fixed in: [371a7c5](https://github.com/TrenchBoot/test-results/pull/1/commits/371a7c522b90b1eeba4e86cabf2f71db202a3398)
Changed test name to "`slaunch-boot` menuentry boots into OS" and "`boot` menuentry boots into OS"

**Review by krystian-hebel — APPROVED** [review]

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**